Generators/AvailableElementsGenerator.cs

```diff
@@ -109,17 +36,19 @@ public void Initialize(GeneratorInitializationContext context)
         /// <inheritdoc/>
         public void Execute(GeneratorExecutionContext context)
         {
-            // Add the ElementFactory model source to compilation. 
-            context.AddSource("Svg_Model", SourceText.From(ModelText, Encoding.UTF8));
+            //// Add the ElementFactory model source to compilation. 
+            //context.AddSource("Svg.Model.g.cs", SourceText.From(SourceTexts.ModelText, Encoding.UTF8));
```

Comment:
Is this not needed?

Reply:
Moved to `Source/Svg.Model.cs`. Initiallly moved the class to template `SourceTexts`, before deleting it from the Generator.

Reply:
Ah. missed this.